Microsoft.Research/Contracts/MsCorlib/System.Threading.Tasks.Task.cs

```diff
@@ -258,7 +258,13 @@ public Task(Action<object> action, object state, CancellationToken cancellationT
     // Returns:
     //     The System.AggregateException that caused the System.Threading.Tasks.Task
     //     to end prematurely.
-    //public AggregateException Exception { get; }
+    public AggregateException Exception
```

Comment:
We can add a postcondition that result is not null if the status is Faulted (but not sure that this would be helpful).